featuretools/primitives/standard/datetime_transform_primitives.py

```diff
@@ -331,6 +331,39 @@ def is_leap_year(vals):
         return is_leap_year
 
 
+class IsLunchTime(TransformPrimitive):
+    """Determines if a datetime falls during configurable lunch hour
+
+    Args:
+        lunch_hour (int): Time when lunch is taken. Defaults to 12
+
+    Examples:
```

Comment:
Also would be good to include a doctest example of how to use the primitive with a custom hour set.